content/en/mobile_app_testing/_index.md

```diff
@@ -23,14 +22,19 @@ cascade:
     tags: ['mobile_testing']
 ---
 
-{{< site-region region="us3,us5,gov,eu,ap1" >}}
-<div class="alert alert-warning">Mobile Application Testing is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< site-region region="us,us5,eu" >}}
+<div class="alert alert-warning">Mobile Application Testing is generally available for US1, US5, and EU sites. Support for US3 and AP1 coming soon.</div>
 {{< /site-region >}}
 
-{{< site-region region="us" >}}
-<div class="alert alert-info">Mobile Application Testing is in limited availability and is only supported for the Datadog US1 site.</div>
+{{< site-region region="us3,ap1" >}}
+<div class="alert alert-warning">Support for Mobile Application Testing is coming soon.</div>
```

Comment:
We only document the product as it is now, without making promises for the future. My suggestion would be to leave this as is until support for US3 and AP1 is available.